cli/f3.go

```diff
@@ -0,0 +1,168 @@
+package cli
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/filecoin-project/go-f3/manifest"
+)
+
+var (
+	F3Cmd = &cli.Command{
+		Name:  "f3",
+		Usage: "Manages Filecoin Fast Finality (F3) interactions",
+		Subcommands: []*cli.Command{
+			{
+				Name:    "list-participants",
```

Comment:
```suggestion
				Name:    "list-miners",
```

Because I'm _guessing_ that's what the user will expect.